### DISCUSSION TITLE
Insufficient write permissions for /config

### DISCUSSION THREAD

**Discussion by JohnnyTendo** [wolveix/satisfactory-server#356] [Q&A] [answered]
Hi,
I'm trying to run this as a container on a remote server of mine, but unfortunately I did not succeed to get it running. I'm using the docker-compose.yml with some minor changes (baisically just the DEBUG env var and some traeffik labels).
Whenever the container starts it generates the following output:

> Recreating satisfactory-server ... done
Attaching to satisfactory-server
satisfactory-server    | ===== Satisfactory Server 1.8.7 =====
satisfactory-server    | https://github.com/wolveix/satisfactory-server
satisfactory-server    | 
satisfactory-server    | Checking available memory: 13GB detected
satisfactory-server    | Clearing old Satisfactory logs (set LOG=true to disable this)
satisfactory-server    | The current user does not have write permissions for /config

I already tried using a clone of the repo and it's dockerfile, to overwrite the init.sh and also to bypass it completely to manually start it while being attached to the container, to get some more information. While being attached to the container, I can freely write to the /config dir with the steam user specified in the init.sh (which is not that easy without any editor in the base image ^^ ).

Docker is not really my home domain, so I'm a bit puzzled what to do next. Do you have any suggestions, what I might've been missing?

**Comment by wolveix** [comment, marked as answer]
Hey @JohnnyTendo, please fill out the issue template and I can help you further :)

**Reply by JohnnyTendo** [reply]
Hey @wolveix . thank you, I created an issue. Thank you. For the all the work & effort you put into this repo and the community :)
